src/System.Runtime.WindowsRuntime/src/System/InternalHelpers.cs

```diff
@@ -43,38 +43,6 @@ internal static void TryDeregister(this CancellationTokenRegistration ctr)
 
     internal class Helpers
     {
-        private static Task s_completedTask = null;
-
-        internal static Task CompletedTask
-        {
-            get
-            {
-                var completedTask = s_completedTask;
-                if (completedTask == null)
-                {
-                    var taskSource = new TaskCompletionSource<VoidValueTypeParameter>();
-                    taskSource.SetResult(default(VoidValueTypeParameter));
-                    s_completedTask = completedTask = taskSource.Task;
-                }
-                return completedTask;
-            }
-        }
-
-        internal static Task<TYPE> TaskFromException<TYPE>(Exception e)
-        {
-            var taskSource = new TaskCompletionSource<TYPE>();
-            taskSource.SetException(e);
-            return taskSource.Task;
-        }
-
-        internal static Task<TYPE> TaskFromCancellation<TYPE>(CancellationToken cancellationToken)
-        {
-            var taskSource = new TaskCompletionSource<TYPE>();
-            taskSource.SetCanceled();
-            taskSource.Task.Wait(cancellationToken);
-            return taskSource.Task;
-        }
-
         internal unsafe static void ZeroMemory(byte* src, long len)
         {
             while (len-- > 0)
```

Comment:
Don't know if its a hot path; but should this operate on a larger size than byte (e.g. long/int) when size is larger?